iothub/service/src/Messaging/StreamDisposalResponsibility.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices
+{
+    // Enum that indicates if the Stream used to initialize an IoT hub message
+    // should be disposed by the application or by the SDK.
+    internal enum StreamDisposalResponsibility
```

Comment:
FYI @tmahmood-microsoft this will go away when you work on removing IDisposable from the Message class.